clang/lib/DPCT/RulesAsm/Parser/AsmTokenKinds.def

```diff
@@ -404,6 +404,12 @@ MODIFIER(idx, ".idx")
 MODIFIER(bfly, ".bfly")
 MODIFIER(sc, ".sc")
 MODIFIER(gl, ".gl")
+MODIFIER(L1, ".L1")
+MODIFIER(L2, ".L2")
+MODIFIER(tensormap, ".tensormap")
+MODIFIER(evict_last, "::evict_last")
+MODIFIER(evict_normal, "::evict_normal")
```

Comment:
We should treat "evict_last", and "evict_normal" as tokens instead of  "::evict_last" and "::evict_normal".

Reply:
Done